File: doc/src/scripts/Vector3.cs
using KRPC.Client;
using KRPC.Client.Services.SpaceCenter;
using System;
using System.Net;

class VectorExample
{
    public static void Main ()
    {
        var connection = new KRPC.Client.Connection ();
        var vessel = connection.SpaceCenter ().ActiveVessel;
        KRPC.Client.Tuple<double,double,double> v = vessel.Flight ().Prograde;
        Console.WriteLine (v.Item1 + "," + v.Item2 + "," + v.Item3);
    }
}

File: doc/src/scripts/Vector3.cpp
#include <krpc.hpp>
#include <krpc/services/space_center.hpp>
#include <iostream>

using namespace krpc;
using namespace krpc::services;

int main() {
  Client conn = krpc::connect();
  SpaceCenter sc(&conn);
  auto v = sc.active_vessel().flight().prograde();
  std::cout << v.get<0>() << " " << v.get<1>() << " " << v.get<2>() << std::endl;
}
